Jenkinsfile

```diff
@@ -188,6 +188,19 @@ pipeline {
 
     post {
         always {
+            node('tester') {
+                deleteDir()
+                unstash 'CIScripts'
+                script {
+                    try {
+                        unstash 'testReport'
+                        powershell script: './CIScripts/GenerateTestReport.ps1'
```

Comment:
What do you think about passing report file name(s) as a parameter(s) to GenerateTestReport?

Reply:
I think it might be a good addition. We'll change when reports from othere testing frameworks will occur.